### DISCUSSION TITLE
Follow up: using zip files af refseq

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse#1633] [Q&A] [answered]
Hello i tried using zipped files as refseq and got the following display

![Capture](https://user-images.githubusercontent.com/91090403/216659469-95dd3a07-9950-4371-9582-5e60faff4b75.PNG)

For information here is my tracks.conf file:
[GENERAL]

highResolutionMode=auto

refSeqs=iwgsc_refseqv2.1_assembly.fa.fai

[tracks.IWGSC_RefSeq2_1]

urlTemplate=iwgsc_refseqv2.1_assembly.fa.gz

storeClass=JBrowse/Store/SeqFeature/IndexedFasta

type=Sequence

category=Reference Sequence

description = IWGSC RefSeq v2.1

Did i do something wrong?

Thanks and sorry for asking so many questions

**Comment by cmdcolin** [comment]
I think you can fix it with `storeClass=JBrowse/Store/SeqFeature/BgzipIndexedFasta`

no worries :)

**Comment by MehdiiYac** [comment, marked as answer]
Hello, i tried that but the sequence keeps loading forever:
![image](https://user-images.githubusercontent.com/91090403/216995008-721fdd1a-c709-4046-b259-7e5540b23323.png)
Do you have any idea of what could be wrong?

**Reply by MehdiiYac** [reply]
@cmdcolin (just in case you didn't see it)

**Reply by cmdcolin** [reply]
@MehdiiYac not exactly sure, thanks for the reminder though. I created a test case where I loaded a genome using a config that looks like yours and it worked ok. here is a config that works for me

```
[GENERAL]
highResolutionMode=auto
refSeqs=iwgsc_refseqv2.1_assembly.fa.gz.fai
[tracks.IWGSC_RefSeq2_1]
urlTemplate=iwgsc_refseqv2.1_assembly.fa.gz
storeClass=JBrowse/Store/SeqFeature/BgzipIndexedFasta
type=Sequence
category=Reference Sequence
description = IWGSC RefSeq v2.1
```

note that i use .gz.fai not just .fai. you should run samtools faidx on the bgzip fasta e.g. samtools faidx file.fa.gz
